sdk/python/feast/infra/offline_stores/contrib/postgres_offline_store/postgres.py

```diff
@@ -214,7 +223,7 @@ def pull_all_from_table_or_query(
 
         query = f"""
             SELECT {field_string}
-            FROM {from_expression} AS paftoq_alias
```

Comment:
i think someone added this in because it caused a crash without this? https://github.com/feast-dev/feast/pull/2956

Reply:
I had a similar issue. I thought it was caused by the fact data_source.get_table_query_string() always puts "()" around the from_expression. Even when from_expression is just a table name. I decided to adjust get_table_query_string() inspired by redshift_source.py. It's rather unfortunate that the context of #2954 isn't clear. because I had the exact same error. But I'm unsure if his fix works for me and vice versa. I think adding AS paftoq_alias and combining it with my version of get_table_query_string() would be the safest option.

Reply:
yeah im not sure. But what you said as the safe option sounds good to me